docs/usermanual.md

```diff
@@ -486,7 +490,7 @@ This command will only work after the developer has executed init-dev-env (that'
 
 This command will also launch the coverage, doc and complexity task (see in the above sections).
 
-## <a name="newtransport"/> New transport development
+## New transport development
```

Comment:
Now the index link to this subsection does not work

Reply:
Fixed - facd80d - ToC regenerated